tests/playbooks/roles/install-cpo-occm/defaults/main.yaml

```diff
@@ -9,4 +9,4 @@ run_e2e: false
 # Used for access the private registry image from k8s
 remote_registry_host: "{{ ansible_default_ipv4.address }}"
 generated_image_url: "{{ remote_registry_host }}/openstack-cloud-controller-manager:v0.0.{{ github_pr }}"
-image_url: "{{ generated_image_url if build_image else 'registry.k8s.io/provider-os/openstack-cloud-controller-manager:v1.27.1' }}"
+image_url: "{{ generated_image_url if build_image else 'registry.k8s.io/provider-os/openstack-cloud-controller-manager:v1.27.2' }}"
```

Comment:
yeah the problem is coming from this row, the cinder e2e test does not build the image - instead tries to use image that does not exist yet. It is kind of chicken - egg problem.

Reply:
https://github.com/kubernetes/cloud-provider-openstack/pull/2351 this tries to fix this issue.